CHANGELOG.md

```diff
@@ -5,11 +5,37 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [5.0.0] - 2019-12-xx
+
+### Features
+- Invalid JSON pointers are reported as errors now [json-ref-resolver#140](https://github.com/stoplightio/json-ref-resolver/pull/140) and [json-ref-resolver#147](https://github.com/stoplightio/json-ref-resolver/pull/147)
+- Properties are iterated in the order they appear in the document [#730](https://github.com/stoplightio/spectral/issues/730)
+- Non-JSON-ish YAML mapping keys are reported [#726](https://github.com/stoplightio/spectral/issues/726)
+- CLI: new formatter - HTML [#389](https://github.com/stoplightio/spectral/issues/389)
+- CLI: new formatter - text [#822](https://github.com/stoplightio/spectral/issues/822)
+- CLI: new formatter - teamcity [#823](https://github.com/stoplightio/spectral/issues/823)
+- CLI: custom resolver can be provided leveraging --resolver flag [#717](https://github.com/stoplightio/spectral/issues/717)
+- CLI: input can be provided via STDIN [#757](https://github.com/stoplightio/spectral/issues/757)
+- Implemented ignoreUnsupportedFormats to make it easier to detect unrecognized formats [#678](https://github.com/stoplightio/spectral/issues/678)
+- Rule's Given can be an array now [#799](https://github.com/stoplightio/spectral/pull/799)
+- Casing built-in function is added [#564](https://github.com/stoplightio/spectral/issues/564)
 
 ### Changed
+- Improved error source detection [#685](https://github.com/stoplightio/spectral/pull/685)
 - Built-in OAS rule `openapi-tags` is now recommended [#706](https://github.com/stoplightio/spectral/pull/706)
 - CLI: Default `--fail-severity` is now `warn`, so getting an `info` or a `hint` will not return a exit status code [#706](https://github.com/stoplightio/spectral/pull/706)
+- When was removed from Rule [#585](https://github.com/stoplightio/spectral/issues/585)
+- Rules are recommended by default [#719](https://github.com/stoplightio/spectral/pull/719)
```

Comment:
```suggestion
- BREAKING: Rules are recommended by default [#719](https://github.com/stoplightio/spectral/pull/719)
```

Maybe put this at the top of the section?

Reply:
I moved all breaking to the top.